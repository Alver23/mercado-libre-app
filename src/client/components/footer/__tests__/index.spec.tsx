// Dependencies
import { render, screen, cleanup } from '@testing-library/react';

// Under test file
import Footer from '../index';

// Mocks
jest.mock('@alversoft/client/core/config', () => ({ siteName: 'fake test' }));

describe('<Footer />', () => {
  afterEach(cleanup);

  it('should render correctly', () => {
    render(<Footer />);
    const linkElement = screen.getAllByText('fake test');
    expect(linkElement).toBeInTheDocument();
  });
});
