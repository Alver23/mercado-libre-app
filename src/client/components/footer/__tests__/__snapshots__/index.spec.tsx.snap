// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should save a snapshot of the component 1`] = `
<footer
  className="flex justify-center items-center bg-white h-12 bg-gray-400"
>
  <p
    className="text-gray-500 text-center text-lg"
  >
    Copyright Â© 1999-
    2021
     
    fake test
     LTDA.
  </p>
</footer>
`;
